skyvern/webeye/scraper/domUtils.js

```diff
@@ -601,6 +618,21 @@ function buildTreeFromBody() {
     } else if (attrs["role"] && attrs["role"].toLowerCase() === "listbox") {
       // if "role" key is inside attrs, then get all the elements with role "option" and get their text
       selectOptions = getListboxOptions(element);
+    } else if (isComboboxDropdown(element)) {
+      // open combobox dropdown to get options
+      element.click();
```

Comment:
very interesting - interacting with the dropdown elements in order to get/parse dropdown elements.

would element ids / unique ids be assigned to the dropdown elements? would line 629 (pressing tab) makes those elements gone. if that happens, then the dropdown element ids we're sending to llm won't be interactable anymore

Reply:
makes sense